packaging/systemd/configure-ovs-microshift.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -eux
+
+OVS_HANDLER_THREADS=${OVS_HANDLER_THREADS:-1}
+OVS_REVALIDATOR_THREADS=${OVS_REVALIDATOR_THREADS:-1}
+
+# let ovn-controller know we are using an unix socket for SBDB communication instead of TLS/TCP
+ovs-vsctl set open . external-ids:ovn-remote="unix:/var/run/ovn/ovnsb_db.sock"
```

Comment:
we probably want to leave this configuration to ovn-kubernetes.

Reply:
Oh, I thought we had to do this, I was following your .md README, I can remove it now, and we can re-add it in your patch if it's necessary